release notes/v0.7.0.md

```diff
@@ -0,0 +1,64 @@
+xk6-browser v0.7.0 is here! :tada:
+
+We've been busy preparing for big changes in the future, so this minor release is smaller than usual. It contains a new feature to unblock users using `dialog` boxes, and internal improvements.
+
+
+## Bugs fixed
+
+- `xk6-browser` no longer panics when it navigates to a website that contains `iframes` which themselves navigate to a different origin to the main `frame` (e.g. a website containing an embedded Youtube video). ([#677](https://github.com/grafana/xk6-browser/pull/677))
+
+
+## New features
+
+- `xk6-browser` will now automatically dismiss dialog boxes (`alert`, `confirm`, `prompt`, and `beforeunload`). ([#663](https://github.com/grafana/xk6-browser/pull/663))
+
+
+## Improvements
+
+- Improved launching of a browser without providing any options. ([#649](https://github.com/grafana/xk6-browser/pull/649))
+ 
+  We couldn't launch a new browser without providing an empty object. We've fixed this, so now you can launch a new browser without an empty object.
+    
+    With the empty object:
+
+    ```js
+    const browser = chromium.launch({});
+    ```
+
+    Without the empty object:
+
+    ```js
+    const browser = chromium.launch();
+    ```
+
+
+### Documentation
+
+- Updated the demo video on our README. ([#639](https://github.com/grafana/xk6-browser/pull/639))
+
+  Now it shows the usage for xk6-browser version v0.6.0 and above.
+
+- Simplified the README. ([#656](https://github.com/grafana/xk6-browser/pull/656))
+
+  You can now find the documentation [here](https://k6.io/docs/javascript-api/xk6-browser/).
+
+- Updated the project roadmap. ([#658](https://github.com/grafana/xk6-browser/pull/658))
+
+
+## Internals
+
+- Upgraded `k6` dependency to `v0.42.0`. ([#?](https://github.com/grafana/xk6-browser/pull/?))
+
+- Refactored the lifecycle events code. ([#647](https://github.com/grafana/xk6-browser/pull/647), [#644](https://github.com/grafana/xk6-browser/pull/644))
+	
+    This shouldn't change any existing behavior, but if you notice anything that doesn't work as expected, please let us know. 
+
+- Fix for CI. ([#674](https://github.com/grafana/xk6-browser/pull/674))
+
+### Experimental Module Merge into k6
+
+- `xk6-browser` is now importable as an experimental module from `k6`. ([#667](https://github.com/grafana/xk6-browser/pull/667))
```

Comment:
I believe this might create wrong impression that `xk6-browser` can now be imported as `"k6/experimental/browser"`, while the PR is only prevents the browser extension from being registered automatically as `k6/x/browser`.

Reply:
Resolved in: 6e87b3094a95af9b5270e7f4e7c42e67e3f7fd3a